test/helpers/getProjectConfig.js

```diff
@@ -0,0 +1,46 @@
+import glob from 'glob';
+import getProjectConfig from '../../src/helpers/getProjectConfig';
+
+jest.mock('glob');
+
+beforeEach(() => {
+  glob.sync = jest.fn();
```

Comment:
I couldn't get `glob` to work normally in my tests, even with very general obvious globs. I was getting a `cannot get length of undefined` error.

Reply:
is there a reason it needs to be mocked at all, since you've got the actual test files committed?

Reply:
I'm able to target the files I need with `glob.sync` using the glob path on [L20](https://github.com/airbnb/react-component-variations/pull/21/files#diff-031de962dd91d0d8a4f21e786ac7dd69R20) `{ projectConfigPath: '*/fixtures/fun-monorepo/*/project.json' }`. However, when this test runs using Jest, `glob.sync` throws with the `cannot get length of undefined` error no matter how general the glob. I think this maybe a bug, any ideas? I can try and make a repro-case.